core/include/cubos/core/memory/opt.hpp

```diff
@@ -128,7 +128,8 @@ namespace cubos::core::ecs
         /// @param other Other value.
         /// @return Whether they're equal.
         template <typename U = T> // See https://brevzin.github.io/c++/2021/11/21/conditional-members/
-        requires(requires(const U& a, const U& b) { a == b; }) bool operator==(const Opt<U>& other) const
+            requires(requires(const U& a, const U& b) { a == b; })
+        bool operator==(const Opt<U>& other) const
```

Comment:
Formatting error

Reply:
again #800 :(

Reply:
If you're using VS Code, you can save without formatting with Ctrl Shift P -> 'save without formatting'